src/components/AccountSecurity/LoginPage.tsx

```diff
@@ -30,10 +30,11 @@ interface Props {
   role: Role,
   alert: any
   autoLogout: boolean, // whether or not the user was logged out automatically
-  setAutoLogout: (boolean) => void // stop showing the logged out automatically banner once user navigates away from the page
+  setAutoLogout: (boolean) => void, // stop showing the logged out automatically banner once user navigates away from the page
+  useRecaptcha: boolean
 }
 
-class LoginPage extends Component<Props, State> {
+export class LoginPage extends Component<Props, State> {
```

Comment:
I need to export the base `LoginPage` class separately from the default export, since the default export is wrapped with the `withAlert` HOC, which causes issues in tests